testassets/InteropTestsClient/RunGrpcWebTests.ps1

```diff
@@ -0,0 +1,28 @@
+﻿Param
+(
+    [bool]$use_tls = $false
+)
+
+$allTests =
+  "grpcweb_unary",
```

Comment:
are these "standard" scenarios that are documented somewhere or we just made those up?
For regular interop tests, we have https://github.com/grpc/grpc/blob/master/doc/interop-test-descriptions.md, I don't recall seeing a similar doc for grpc-web.
We should reach out to grpc-web team to see what's the best way to test against other implementations (="interop tests")

Reply:
These are from https://github.com/johanbrandhorst/grpc-web-compatibility-test. I've added a comment.